source/containerimage/pull.go

```diff
@@ -222,7 +222,7 @@ func getLayers(ctx context.Context, provider content.Provider, desc ocispec.Desc
 		return nil, errors.Wrap(err, "failed to unmarshal manifest")
```

Comment:
```
	manifest, err := images.Manifest(ctx, provider, desc, platforms.Format(platforms.Default()))
	if err != nil {
		return nil, errors.WithStack(err)
	}
```
should fix the ci. I think `util.imageutil` has same problem but that can be left for another pr.